ds_judgements_public_ui/templates/includes/facets/year_search_facets.html

```diff
@@ -0,0 +1,8 @@
+{% load query_filters court_utils %}
```

Comment:
check if you need both of these (in multiple places)

Reply:
It looks like we do, alas.